src/mito2/src/config.rs

```diff
@@ -84,6 +84,10 @@ pub struct MitoConfig {
     pub vector_cache_size: ReadableSize,
     /// Cache size for pages of SST row groups. Setting it to 0 to disable the cache.
     pub page_cache_size: ReadableSize,
+    /// Cache size for metadata of inverted index. Setting it to 0 to disable the cache.
+    pub inverted_index_metadata_cache_size: ReadableSize,
+    /// Cache size for inverted index content. Setting it to 0 to disable the cache.
+    pub inverted_index_cache_size: ReadableSize,
```

Comment:
How about putting them in `InvertedIndexConfig`?

Reply:
> How about putting them in `InvertedIndexConfig`?

fixed in 9b457d930